Assets/Scripts/Player/PlayerController.cs

```diff
@@ -73,10 +73,24 @@ public class PlayerController : MonoBehaviour {
             }
         }
     }
+    
+    private bool isClimbing = false;
```

Comment:
Seems to me like isClimbing == activeLadder != null, so we could get rid of isClimbing. Or / and maybe replace it with a Getter like for IsSliding

`
private bool IsClimbing {
    get {
        return activeLadder != null && ladderCooldownTimer <= 0.0f;
    }
}
`

Could then theoretically use the same Getter to get rid of the if-else in Update and move it to the specific methods like ExecuteLadderMove and ExecuteMove. Not sure if that is prettier though.